### PR TITLE
ci: fix lint error

### PR DESCRIPTION
**Pull request by 100gle**

### CODE REVIEW

File: backend/src/module/core/__init__.py [hunk 1]
```diff
@@ -1 +0,0 @@
-from .program import Program
```
Comment (zthxxx) [inline backend/src/module/core/__init__.py:1]: it seems used in `backend/src/module/api/program.py`

```python
from module.core import Program
```

there appear to be similar issues with other changes in `__init__.py` as well

File: backend/pyproject.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [tool.ruff]
 # Enable pycodestyle (`E`) and Pyflakes (`F`) codes by default.
 select = ["E", "F", "I"]
-ignore = ['E501']
+ignore = ['E501', 'F401']
```
Comment (zthxxx) [inline backend/pyproject.toml:4]: lint 规则一般人挺难读懂的，能不能把每项规则的连接也贴到注释里？

比如

```yaml
# E501  https://beta.ruff.rs/docs/rules/line-too-long
# F401  https://beta.ruff.rs/docs/rules/unused-import
```

**Comment by zthxxx** [comment]
等 ci test 修复完了再合咯

**Comment by EstrellaXD** [comment]
Approved

**Pull request merged**